client/src/components/graphs/SVGCanvas.js

```diff
@@ -1,19 +1,81 @@
-import React from "react"
 import PropTypes from "prop-types"
-import * as d3 from "d3"
+import React from "react"
+import { Button } from "react-bootstrap"
+import DOWNLOAD_ICON from "resources/download.png"
 
-export default class SVGCanvas extends React.Component {
-  static propTypes = {
-    size: PropTypes.object.isRequired
+const SVGCanvas = props => {
```

Comment:
Prefer `const SVGCanvas = ({ width, …etc… }) => {` instead of referring to `props.something` everywhere.